### PR TITLE
[NUI] Fix the crash issue when IndicatorSize is null(Pagination).

### PR DESCRIPTION
**Pull request by AchoWang**
### Description of Change ###
If IndicationSize is null,  CreateIndicator() will crash.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by elishateng — APPROVED** [review]
LGTM

**Review by zhouleonlei — APPROVED** [review]
LGTM

**Review by zg2nets — APPROVED** [review]
looks good

**Comment by rabbitfor** [comment]
In my opinion, most properties in Style class is not by default. So it is normal that the PaginationStyle.IndicatorSize is null.

How about fixing Pagination.CreateIndicator?

**Comment by dongsug-song** [comment]
@AchoWang 
Hello, could you check the @rabbitfor 's comment?

**Comment by AchoWang** [comment]
> In my opinion, most properties in Style class are null by default. So it is normal that the PaginationStyle.IndicatorSize is null. (And it is better to keep null when the user is not set the property in Style for the future)
> 
> How about fixing Pagination.CreateIndicator?

**Comment by AchoWang** [comment]
> In my opinion, most properties in Style class are null by default. So it is normal that the PaginationStyle.IndicatorSize is null. (And it is better to keep null when the user is not set the property in Style for the future)
> 
> How about fixing Pagination.CreateIndicator?

Done, plz check again, thanks.

**Comment by AchoWang** [comment]
> @AchoWang
> Hello, could you check the @rabbitfor 's comment?

Done.

File: src/Tizen.NUI.Components/Controls/Pagination.cs [hunk 1]
```diff
@@ -337,6 +337,10 @@ public int SelectedIndex
             }
             set
             {
+                if (paginationStyle.IndicatorSize == null)
```
Comment (rabbitfor) [inline src/Tizen.NUI.Components/Controls/Pagination.cs:340]: I think the following case won't work correctly.
```
new Pagination()
{
    SelectedIndex = 2,
    IndicatorCount = 3,
    IndicatorSize = new Size(10, 10)
}
```

**Review by rabbitfor — APPROVED** [review]
LGTM

**Pull request merged**